io/fv3atm_history_io.F90

```diff
@@ -187,8 +187,7 @@ subroutine history_type_register(hist, Diag, Time, Atm_block, Model, xlon, xlat,
     hist%fhzero = Model%fhzero
     !   hist%ncld   = Model%ncld
     hist%ncld   = Model%imp_physics
-    hist%nsoil  = Model%lsoil
-    hist%nsoil_lsm  = Model%lsoil_lsm
+    hist%nsoil  = Model%lsoil_lsm
```

Comment:
@JiliDong-NOAA Please take a look here. I noticed that there was a difference between what exists in the develop branch and your PR into production/RRFS.v1 here: https://github.com/NOAA-EMC/fv3atm/pull/869. I'm not sure that I handled this conflict correctly.